src/commissaire/models/__init__.py

```diff
@@ -241,6 +241,11 @@ def _coerce(self):
 class Network(Model):
     """
     Representation of a network.
+
+    .. note::
+
+       This model is similar to ContainerManagerConfig. The options
+       attribute holds configuration items used to define network options.
```

Comment:
It does have an ``options`` attribute once it's been instantiated, but I'll go ahead and remove it to avoid confusion. :+1: